app/presenters/case_history.rb

```diff
@@ -33,16 +33,24 @@ def event
     end
   end
 
-  def previous_primary_pom_id
-    @previous_allocation.primary_pom_nomis_id
+  def primary_pom_email
+    HmppsApi::PrisonApi::PrisonOffenderManagerApi.fetch_email_addresses(@allocation.primary_pom_nomis_id).first
   end
 
-  def previous_primary_pom_name
-    @previous_allocation.primary_pom_name
+  def secondary_pom_email
+    HmppsApi::PrisonApi::PrisonOffenderManagerApi.fetch_email_addresses(@allocation.secondary_pom_nomis_id).first
+  end
```

Comment:
It might be worth dropping a small comment in to acknowledge that fetching data in the View layer is a 'Bad Thing', but it gets us one step closer to our target data model.

The POM's email address is something which should belong to the data object itself, rather than being injected by the Controller or View. We'll be able to do this once our data model is a bit smarter.